src/components/Card/Card.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Button from "../Button/Button";
+import "./Card.css"
+
+function Card({ fruit, amount, setAmount }) {
+
+    function counter(operator) {
+        setAmount((prev) => (prev + operator >= 0 ? prev + operator : 0));
+    }
+
+    return (
+        <div className="card">
+            <h1 className="card-header">{fruit}</h1>
+            <div className="inner-container-card">
+                <Button
```

Comment:
Ik zie liever dat er door middel van een `disabled` property de button uitgezet wordt wanneer de counter `0` is. Dan hoef je ook geen aparte functie te maken die ervoor zorgt dat de counter niet ónder de 0 gaat